packages/app-desktop/app.ts

```
@@ -212,7 +212,12 @@ class Application extends BaseApplication {
 			const contextMenu = Menu.buildFromTemplate([
 				{ label: _('Open %s', app.electronApp().name), click: () => { app.mainWindow().show(); } },
 				{ type: 'separator' },
-				{ label: _('Quit'), click: () => { void app.quit(); } },
+				{ label: _('Quit'), click: () => {
+					app.quitWithSyncCheck(
+						(action: any) => this.store().dispatch(action),
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] A space is required before '}'.

(`@typescript-eslint/object-curly-spacing`)

---

[error] A space is required before '}'.

(object-curly-spacing)